bundle/credentials.go

```diff
@@ -0,0 +1,8 @@
+package bundle
+
+// Credential represents the definition of a CNAB credential
+type Credential struct {
+	Location    `mapstructure:",squash"`
```

Comment:
lol this sent me off on a rabbit hole of "huh, inlining doesn't use 'squash'..." then I found this for the mapstructure lib used by viper https://github.com/mitchellh/mapstructure/issues/42. It got my vote! 👍